### ISSUE TITLE
Problems with SARL project Natures

### ISSUE THREAD

**Issue by dlizondo** [sarl/sarl#474]
In Pre-Configured Eclipse with SARL SDK version 0.3.1 (stable) for Linux x64 (Ubuntu), the project Natures for SARL doesn't appears. In simple words, it is just an Eclipse simple editor with Java Environment. For example, I can't create a Sarl project or anything related to Sarl.

Regards,

Pedro and Diego.

**Comment by gallandarakhneorg** [comment]
Have you created the SARL project with `File > New > SARL Project` or with Maven?

Nevertheless, for enabling the SARL nature by hand (menus will be available in version 0.4), you could add  into your `.project` file the following lines.
```xml
	<buildSpec>
		<buildCommand>
			<name>org.eclipse.xtext.ui.shared.xtextBuilder</name>
			<arguments>
			</arguments>
		</buildCommand>
		<buildCommand>
			<name>org.eclipse.jdt.core.javabuilder</name>
			<arguments>
			</arguments>
		</buildCommand>
	</buildSpec>
	<natures>
		<nature>io.sarl.eclipse.SARLProjectNature</nature>
		<nature>org.eclipse.xtext.ui.shared.xtextNature</nature>
		<nature>org.eclipse.jdt.core.javanature</nature>
	</natures>
```

**Comment by gallandarakhneorg** [comment]
Please try the unstable version (version 0.4-20160908 or higher) that is planned to be released during the next couple of weeks.

**Comment by gallandarakhneorg** [comment]
This issue seems to be solved in version 0.4.
